reposerver/repository/repository.go

```diff
@@ -1023,9 +1023,9 @@ func runHelmBuild(appPath string, h helm.Helm) error {
 	manifestGenerateLock.Lock(appPath)
 	defer manifestGenerateLock.Unlock(appPath)
 
-	// the `helm dependency build` is potentially time consuming 1~2 seconds
-	// marker file is used to check if command already run to avoid running it again unnecessary
-	// file is removed when repository re-initialized (e.g. when another commit is processed)
+	// the `helm dependency build` is potentially a time-consuming 1~2 seconds,
```

Comment:
nit: `a time` -> `time`

Reply:
Hmm, perhaps 

```suggestion
	// the `helm dependency build` potentially takes 1-2 seconds,
```

?